infra/README.md

```diff
@@ -0,0 +1,63 @@
+# Meshdb Environment Setup
+
+These instructions will set up a 4 node k3s cluster on proxmox.
+- 1 "manager" node for control plane and to be used for deployments.
+- 3 "agent" nodes to run services.
+
+1. Configure a user for the [proxmox provider](https://registry.terraform.io/providers/Telmate/proxmox/latest/docs) and set up env vars.
+2. Setup tfvars + ssh keys
+```
+cd meshdb/infra/tf/
+cp example.tfvars your_env.tfvars
+# Modify your_env.tfvars to meet your needs
+```
```

Comment:
```suggestion
2. Setup tfvars + ssh keys
```
ssh-keygen -t ed25519 -f <path_to_meshdb>/meshdb/infra/tf/<environment_name>

cd meshdb/infra/tf/
cp example.tfvars your_env.tfvars
# Modify your_env.tfvars to meet your needs
```
```